jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -156,7 +156,7 @@ pipeline {
                         BUILD_DOCKER = 'build_docker_with_build_number_tag'
                     }
                     def inputManifestObj = lib.jenkins.InputManifest.new(readYaml(file: "manifests/${INPUT_MANIFEST}"))
-                    env.version = inputManifestObj.build.version
+                    env.version = inputManifestObj.build.version + (inputManifestObj.build?.qualifier ? "-${inputManifestObj.build.qualifier}" : "")
```

Comment:
You should change the part in validation, not on build, because build can correctly handle the version and qualifier due to input manifest have both fields.

Reply:
Or if this is build not correctly sending version to validation lib, you should change the part in lib.